src/collector.js

```diff
@@ -76,6 +77,10 @@ export default class Collector {
     return false;
   }
 
+  recordScannedFile(filename) {
+    this.scannedFiles.push(filename);
```

Comment:
Presumably with some of the scanner running over everything and some runnning over a subset this will have dupes?